app/v2/broadcast/post_broadcast.py

```diff
@@ -3,9 +3,16 @@
 from flask import current_app, jsonify, request
 from notifications_utils.polygons import Polygons
 from notifications_utils.template import BroadcastMessageTemplate
+from sqlalchemy.orm.exc import NoResultFound
 
 from app import api_user, authenticated_service
+from app.broadcast_message.rest import (
```

Comment:
That is a good point, I did think about moving it out, was not sure what the best place would be, dao file or a new utils file in broadcast_message, and worried we would end up duplicating tests even more, so ended up leaving it where it was.

Still, you are right we don't usually import from rest and could have tests more at source. I will move it to a utils file I think.

Reply:
Ok, this has been moved to utils now, and test have been moved and adjusted.

Reply:
Great stuff - that's a lot of tests moved 🏅. I like the balance we now have: one file with a few tests for the actual requests + another file with lots of other tests for edge cases and details. Calling the function directly is also simpler than figuring out what request to make, so this will make it easier to extend in future.